serverless/sql-databases/how-to/connect-to-a-database.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to connect to a Serverless SQL Database
+  description: This page explains how to set up a connection to a Serverless SQL Database
+content:
+  h1: How to connect to a Serverless SQL Database
+  paragraph: This page explains how to set up a connection to a Serverless SQL Database
+tags: database serverless-sql-database serverless-db serverless-database postgresql 
+dates:
+  validation: 2023-07-19
+  posted: 2021-07-19
+categories:
+  - serverless
+  - sql-databases
+---
+
+This page shows you how to set up the connection to a Serverless SQL Database using the Scaleway Console.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have created a [Serverless SQL Database](/serverless/sql-databases/how-to/create-a-database/)
+</Message>
+
+## How to set up credentials
+
+1. From the **Overview** tab of your database, click the **Set up credentials** button. The setup wizard displays.
+
+2. Select the [principal](/identity-and-access-management/iam/concepts/#principal) to connect to the database:
+
+    - Yourself, as an [IAM user](/identity-and-access-management/iam/concepts/#user).
+
+    - An [application](/identity-and-access-management/iam/concepts/#application) that you [previously created](/identity-and-access-management/iam/how-to/create-application/) in the IAM section of the console.
+
+    <Message type="note">
+        Refer to the [IAM documentation](/identity-and-access-management/iam/reference-content/overview/) for more information on identities and access rights.
+    </Message>
+
+3. Set up the **connection string** or **connection parameters**:
+
+    - If you already have a secret key, paste it in the connection text in place of the ``[INSERT_SECRET_KEY_HERE]`` placeholder.
+
+    - If you do not have a secret key, click **Generate new secret key**.
+
+4. Copy your connection string or parameter and store it securely.
```

Comment:
It seems like there's a part of this page missing which explains how to actually use your secret key to connect to the DB?

Reply:
It depends a lot on the environment, there are several ways to access a DB and perform operations in it, but it seems pretty obvious for developpers an engineers

Reply:
procedure for user added, generic message for applications has been added